ods_ci/run_robot_test.sh

```diff
@@ -22,7 +22,7 @@ EMAIL_SERVER_SSL=false
 EMAIL_SERVER_UNSECURE=false
 OPEN_REPORT_IN_BROWSER=false
 REPORT_BROWSER="firefox" # Default browser to open reports in
-SUBFOLDER=false
+SUBFOLDER=true
```

Comment:
This change should not be in the PR

Reply:
Resolved

Reply:
Hi Raghul, I can see you commented that you resolved some of the concerns here, although, I don't see any change on this PR. Could you assure you have pushed your latest changes so it propagates here? Thanks! :slightly_smiling_face: